JS-Algorithms/challenges.js

```diff
@@ -14,6 +14,14 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 
 const readableTime = (seconds) => {
```

Comment:
In this solution you can also use [Remainder assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder_assignment) in case you're curious how it works